### ISSUE TITLE
Can't install Pandas package 

### ISSUE THREAD

**Issue by tumikosha** [qpython-android/qpython.org#118]
Can't install Pandas package because 
Compression error, bz2 module is not available

You need to build python with BZIP2 support.
Install the following package before building python:
Red Hat/Fedora/CentOS: yum install bzip2-devel
Debian/Ubuntu: sudo apt-get install libbz2-dev

**Comment by riverfor** [comment]
Thanks for feedback,

But we need to cross-compile bzip2 with android's NDK toolchain, it's a little complex. But we will try our best to make it.

**Comment by rmunozr** [comment]
I have the same issue, I cant install Pandas for Qpython, the same with Scipy and other libraries...Is there any way to install them all in one time like Anaconda Launcher ??

**Comment by jandechent** [comment]
Also same issue, would be great to see pandas/numpy support!

**Comment by hansendm** [comment]
Yes very much so. I'm trying to build a qpython vr desktop environment on Android so i can work while traveling. Please post updates frequently.
